Transformations/ShapeChange/ontologyPatcher.py

```diff
@@ -0,0 +1,167 @@
+import logging
+import argparse
+from rdflib import Graph
+
+def main():
+    # initialize command line arguments
+    parser = argparse.ArgumentParser()
+    parser.add_argument('input_file',
+                         help='Specify the input file')
+    parser.add_argument('output_file',
+                         help='Specify the output file')
+    parser.add_argument('--input_format',
+                         default='ttl',
+                         help='Specify the RDFLib compliant input file format')
+    parser.add_argument('--output_format',
+                         default='ttl',
+                         help='Specify the RDFLib compliant output file format')
+    parser.add_argument('--log', '-l',
+                         default='output.log',
+                         help='Specify the logging file')
+    args = parser.parse_args()
+    
+    patcher = ontologyPatcher(args.input_file, args.input_format, args.log)
+    patcher.patchOnClass()
+    patcher.patchDatatypeEnumeration()
+    patcher.patchObjectProperties()
+    patcher.outputGraph(args.output_file, args.output_format)
+
+
+class ontologyPatcher():
+    """A tool for patching ontologies created using ShapeChange.
```

Comment:
Why do we require this patching?

Reply:
added precision in [readme](https://github.com/VCityTeam/UD-Graph/tree/rules/Transformations/ShapeChange#readme)
and patcher [functions](https://github.com/VCityTeam/UD-Graph/blob/aa846ff115fd7ce7eb5a3348a11464d4a3b35351/Transformations/ShapeChange/ontologyPatcher.py#L50)